src/Request/Request.php

```diff
@@ -114,7 +115,7 @@ public function getUrl()
      */
     public function getQuery()
     {
-        return $this->query;
+        return http_build_query($this->query);
```

Comment:
I have mixed feeling to that. Its not bad, just I think it maybe should be done in Client.php ? To not change what this method is returning?